ament_cmake_python/ament_cmake_python-extras.cmake

```diff
@@ -18,7 +18,8 @@ macro(_ament_cmake_python_get_python_install_dir)
     set(_python_code
       "from distutils.sysconfig import get_python_lib"
       "from os.path import relpath"
-      "print(relpath(get_python_lib(prefix='${CMAKE_INSTALL_PREFIX}'), start='${CMAKE_INSTALL_PREFIX}'))"
+      "from binascii import unhexlify"
+      "print(relpath(get_python_lib(prefix='${CMAKE_INSTALL_PREFIX}'), start='${CMAKE_INSTALL_PREFIX}').replace(unhexlify(b'5c').decode(), '/'))"
```

Comment:
Ok, I think the `\\\\` works, maybe I accidentally did three `\\\`. I'll update the commit.